src/kubernetes.cc

```diff
@@ -72,10 +73,11 @@ constexpr const char kKubernetesSchemaNameFormat[] =
 constexpr const char kClusterFullNameFormat[] =
     "//container.googleapis.com/projects/{{project_id}}/{{location_type}}/"
     "{{location}}/clusters/{{cluster_name}}";
-constexpr const char kNodeFullNameFormat[] =
-    "{{cluster_full_name}}/k8s/nodes/{{node_name}}";
-constexpr const char kPodFullNameFormat[] =
-    "{{cluster_full_name}}/k8s/namespaces/{{namespace_name}}/pods/{{pod_name}}";
+constexpr const char kK8sCoreGroupResourceTypeFormat[] = "io.k8s.{{kind}}";
```

Comment:
Why do we use kubernetes.io in some places, and k8s.io in other places? Can we push for consistency throughout, or start a doc that explains why we vary from one place to another? I know we do k8s_ for the monitored resources, but the schema is kubernetes.io for example.

Reply:
Currently, it is the metric names that use kubernetes.io, and the resource types and names use k8s. As long as all references to resources (including monitored resources) use k8s, it should be fine.

Given Kubernetes has two domains, the naming is going to be split to some extent.

Reply:
I like Bryan's idea of starting a doc to keep track of this, though...